src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -605,19 +605,21 @@ private function registerHttpCacheConfiguration(ContainerBuilder $container, arr
 
         $loader->load('http_cache_purger.xml');
 
-        $definitions = [];
-        foreach ($config['http_cache']['invalidation']['varnish_urls'] as $key => $url) {
-            $definition = new Definition(ScopingHttpClient::class, [new Reference('http_client'), $url, ['base_uri' => $url] + $config['http_cache']['invalidation']['request_options']]);
-            $definition->setFactory([ScopingHttpClient::class, 'forBaseUri']);
+        foreach ($config['http_cache']['invalidatoin']['scoped_clients'] as $client) {
```

Comment:
```suggestion
        foreach ($config['http_cache']['invalidation']['scoped_clients'] as $client) {
```

Reply:
Static analysis should have caught that one?